### DISCUSSION TITLE
The payment status is missing from the CSV file of submissions

### DISCUSSION THREAD

**Discussion by LoriWJ** [solspace/craft-freeform#736] [Questions & Answers]
**Describe the bug or issue you're experiencing**

**Steps to reproduce**

1. Log in to the Craft control panel.
2. Open the list of submissions (Freeform > Submissions). 
3. Select the checkbox next to a submission for a form that has the Stripe payment integration.
4. Select the cog icon that appears at the top of the list and select ‘Export to CSV’.
5. The CSV file doesn't include the payment status column.

**Expected behavior**
The CSV file includes the payment status column as well as the pi_string.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 3.7.63.1
- Freeform Version: 3.13.22.1
- Freeform Edition: pro
- Fresh Install or Upgrade: upgrade

**Comment by kjmartens** [comment]
Hi @LoriWJ,

I think this exporting option is fairly limited. Are you able to use the **Quick Export** option instead? This one will include all 4 Payment-related options, including `Payment Status` (and the ability to set which fields to export). 🙂

**Comment by LoriWJ** [comment]
Thanks for your reply, I appreciate it. 

The finance assistant would prefer to select the individual submissions but will use the 'Quick Export' button or an export profile if it gives her what she wants. But the problem is that the CSV file that these export doesn't include the pi_string in the CSV file, which she needs for reconciliation with Stripe. 

Selecting the individual submissions and using 'Export to CSV' does include the pi_string but not the payment status – which seems like a bug to me.
